test/test_RedisConnection.js

```diff
@@ -108,6 +108,38 @@ builder.add(function testRedisConnection(test) {
   cacheInstance.connect()
 })
 
+builder.add(function testIncr(test) {
+  var cacheInstance = new zcache.RedisConnection('localhost', 6379)
+
+  cacheInstance.on('connect', function () {
+    cacheInstance.removeAllListeners('connect')
+    test.equal(cacheInstance.isAvailable(), true, 'Connection should be available')
+
+    cacheInstance.incr('counter', 15)
+      .then(function () {
+        return cacheInstance.incr('counter')
+      })
+      .then(function (val) {
+        return cacheInstance.get('counter')
```

Comment:
I am.  This is the same test case as for the InMemoryCache, but for Redis.  See line 118.